pkg/sfu/downtrack.go

```diff
@@ -388,7 +392,7 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int) int {
 			size = RTPPaddingMaxPayloadSize + RTPPaddingEstimatedHeaderSize
 		}
 
-		sn, ts, err := d.munger.UpdateAndGetPaddingSnTs(false)
+		sn, ts, err := d.munger.UpdateAndGetPaddingSnTs(d.TargetSpatialLayer() == InvalidSpatialLayer)
```

Comment:
Want to send padding while probing. Force frame marker if stream is paused as it will get a key frame on restart anyway.

Reply:
Will add it to code David.